docs/function_guides/generator.rst

```diff
@@ -165,6 +161,28 @@ by a user function, otherwise it will be ignored.
 The :doc:`Borehole Calibration tutorial<../tutorials/calib_cancel_tutorial>` gives an example
 of the capability to cancel pending simulations.
 
+Modification of existing points
+-------------------------------
+
+To change existing fields of the history array, the generator can initialize an output
+array where the *dtype* contains the ``sim_id`` and the fields to be modified (in
+place of ``gen_specs["out"]``), and then send to the manager as with regular
+communications. Any such message received by the manager will modify the given fields
+for the given *sim_ids*. If the changes are separate from the generation of new points
+and should not mark the generator as ready to receive new points, then send with the
```

Comment:
maybe instead of "If the changes are separate from the generation of new points
and should not mark the generator as ready to receive new points,"

"Since these changes don't correspond with newly generated points, then the generator needs to communicate to the manager that it isn't necessarily ready to receive completed evaluations. Send to the manager with ``keep_state`` set to True."

or something like this

Reply:
@jlnav Shouldn't the first "Since" be an "If"?

Reply:
Updated